src/Vex/PipelineStateCache.cpp

```diff
@@ -32,12 +32,12 @@ const RHIGraphicsPipelineState* PipelineStateCache::GetGraphicsPipelineState(con
     RHIGraphicsPipelineState* ps;
     if (graphicsPSCache.contains(key))
```

Comment:
Now that I think about it, this should be `find`, then either dereference the find result or create and insert.